qcodes_contrib_drivers/drivers/Oxford/Triton.py

```diff
@@ -0,0 +1,204 @@
+                           docstring='Pressure of the condensation line',)
+
+        self.add_parameter(name='pressure_mixture_tank',
+                           unit='Bar',
+                           get_parser=float,
+                           get_cmd=lambda: self.get_pressure('tank'),
+                           docstring='Pressure of the mixture tank',)
+
+        self.add_parameter(name='pressure_forepump_back',
+                           unit='Bar',
+                           get_parser=float,
+                           get_cmd=lambda: self.get_pressure('forepump'),
+                           docstring='Pressure of the forepump back',)
+
+        self.add_parameter(name='temperature_50k_plate',
+                           unit='K',
+                           get_parser=float,
+                           get_cmd=lambda: self.get_temperature('50k'),
+                           docstring='Temperature of the 50K plate',)
+
+        self.add_parameter(name='temperature_4k_plate',
+                           unit='K',
+                           get_parser=float,
+                           get_cmd=lambda: self.get_temperature('4k'),
+                           docstring='Temperature of the 4K plate',)
+
+        if magnet:
+            self.add_parameter(name='temperature_magnet',
+                               unit='K',
+                               get_parser=float,
+                               get_cmd=lambda: self.get_temperature('magnet'),
+                               docstring='Temperature of the magnet',)
+
+        self.add_parameter(name='temperature_still',
+                           unit='K',
+                           get_parser=float,
+                           get_cmd=lambda: self.get_temperature('still'),
+                           docstring='Temperature of the still',)
+
+        self.add_parameter(name='temperature_100mk',
+                           unit='K',
+                           get_parser=float,
+                           get_cmd=lambda: self.get_temperature('100mk'),
+                           docstring='Temperature of the 100mk plate',)
+
+        self.add_parameter(name='temperature_mixing_chamber',
+                           unit='K',
+                           get_parser=float,
+                           get_cmd=lambda: self.get_temperature('mc'),
+                           docstring='Temperature of the mixing chamber',)
+        
+        self.connect_message()
+
+    def vcl2csv(self) -> str:
+        """
+        Convert vcl file into csv file using proprietary binary exe.
+        The executable is called through the python subprocess library.
+        To avoid to frequent file conversion, a timer of self.conversion_timer
+        second is used.
+
+        Returns:
+            str: The output of the bash command
+        """
+        
+        conversion = False
+        if self._timer+self.rr <= time.time():
```

Comment:
Check if self.rr is correct.

Reply:
I have no idea how this happened, corrected now.